docs/en/latest/tutorials/how-to-access-Apache-APISIX-Prometheus-Metrics-on-k8s.md

```diff
@@ -21,24 +26,21 @@ title: How to access Apache APISIX Prometheus metrics on Kubernetes
 #
 -->
 
-Observability (monitoring functionality) has always played an essential role in system maintenance. A sound monitoring system can help engineers quickly understand the status of services running in production environments and locate problems or give early warning of anomalies when they occur.
+This tutorial explains how you can export APISIX metrics in Prometheus format.
 
-*Prometheus* is a leading open-source project focused on metrics and alerting that has changed the way the world does monitoring and observability. For more information, see *Prometheus*'s official [website](https://prometheus.io/).
+:::note
 
-## Before you begin
+This tutorial requires APISIX version > 2.13.
 
-In the APISIX Ingress environment, ensure that the `public-api` and `prometheus` plugins are enabled and the [prometheus pluginAttrs](https://apisix.apache.org/docs/apisix/plugins/prometheus/#specifying-metrics) are configured. Please refer to the following install example:
+:::
 
-```sh
-helm repo add apisix https://charts.apiseven.com
-helm repo update
-helm install apisix apisix/apisix -f values.yaml --create-namespace -n ingress-apisix
-```
+## Enable the prometheus Plugin
 
-<details>
-  <summary>values.yaml</summary>
+First, you have to enable the [prometheus](https://apisix.apache.org/docs/apisix/plugins/prometheus) Plugin. You can do this by adding to your `values.yaml` file while you install APISIX Ingress via Helm. You can also enable the [public-api](https://apisix.apache.org/docs/apisix/next/plugins/public-api) Plugin to expose these metrics.
```

Comment:
Can we remove the "next" prefix in documentation url?

Reply:
Yes. I always forget it.